resources/views/activities/index.blade.php

```diff
@@ -0,0 +1,107 @@
+@extends('layouts.app')
+
+@section('title', __('main.users'))
+
+@section('content')
+
+    <!-- Alerts - OPEN -->
+    @include('parts.alerts')
+    <!-- Alerts - CLOSE -->
+
+    <!-- Language for dates - OPEN -->
+    @php
+        \Date::setLocale(Session::get('locale'));
+    @endphp
+    <!-- Language for dates - CLOSE -->
+
+    <!-- Content - OPEN -->
+    <div class="container margin-top padding-bottom">
+
+        <div class="row">
+            <div class="col-sm"></div>
+            <div class="col-sm">
+                <h3 class="margin-bottom-sm text-center">
+                    {{ __('main.activity_log') }}
+                </h3>
+            </div>
+            <div class="col-sm text-right">
+            <a href="{{ route('activities_delete_all') }}">     <button type="button" class="btn btn-danger"> <span style="padding-right:0.5em;" class="octicon octicon-trashcan"></span> Clear Log</button></a>
+            </div>
+            {{-- <div class="col-sm text-right">
```

Comment:
If not used just remove this comment code. Comments are only to understand the code block.

Reply:
Commented code removed.